tests/baselines/reference/callbackTag1.types

```diff
@@ -10,7 +10,7 @@ var x = 1
 /** @type {Sid} smallId */
 var sid = s => s + "!";
 >sid : Sid
->s => s + "!" : (s: string) => string
+>s => s + "!" : (s: string) => any

```

Comment:
This type doesn't seem right; `Sid` is a callback that returns a string.

Reply:
My bad, that experiment failed. Sorry for the noise! I've dropped that commit.